pkg/virt-operator/kubevirt_test.go

```diff
@@ -2624,6 +2627,71 @@ var _ = Describe("KubeVirt Operator", func() {
 			Expect(kvTestData.resourceChanges["daemonsets"][Patched]).To(Equal(0))           // namespace unpatched
 		})
 
+		It("should apply virt-api replica count from a CustomizeComponents patch", func() {
+			kvTestData := KubeVirtTestData{}
+			kvTestData.BeforeTest()
+			defer kvTestData.AfterTest()
```

Comment:
I think It would be safe to use `DeferCleanup`; e.g.
```suggestion
			DeferCleanup(kvTestData.AfterTest)
```

Reply:
sure